lib/modules/dosomething/dosomething_global/dosomething_global.module

```diff
@@ -203,7 +203,7 @@ function dosomething_global_node_insert($node) {
 /**
  * Implements hook_form_alter().
  */
-function dosomething_global_form_node_form_alter(&$form, &$form_state, $form_id) {
+function dosomething_global_form_node_form_alter(&$form, &$form_state) {
```

Comment:
Hmm, I don't think it should make a difference (the variable is neither used nor returned), but happy to add it back in.